packages/react-meteor-data/README.md

```diff
@@ -328,13 +332,109 @@ const docs = useFind(() => {
+
+The important thing to understand is that your reactive function can be initially called more than once for a single
+render, because sometimes the work will be tossed. Additionally, `useTracker` will not call your reactive function
+reactively until the render is committed (until `useEffect` runs). If you have a particularly fast changing data source,
+this is worth understanding. With this very short possible suspension, there are checks in place to make sure the
+eventual result is always up to date with the current state of the reactive function. Once the render is "committed",
+and the component mounted, the computation is kept running, and everything will run as expected.
+
+
+## Suspendable version of hooks
+
+### `suspense/useTracker`
+
+This is a version of `useTracker` that can be used with React Suspense. It is similar to `useTracker`, 
+but it throws a promise and suspends the rendering until the promise is resolved. Also in this version it has the need
+for a key as its first parameters. the key is used to identify the computation and to avoid recreating it when the 
+component is re-rendered.
+
+```jsx
+import { useTracker } from 'meteor/react-meteor-data/suspense'
+
+function Tasks() { // this component will suspend
+  useSubscribe("tasks");
+  const { username } = useTracker("user", () => Meteor.user())  // Meteor.user() is now async
+  const tasksByUser = useTracker("tasksByUser", () =>
+          TasksCollection.find({username}, { sort: { createdAt: -1 } }).fetchAsync() // fetch will be async
+  );
+
+
+  // render the tasks
+}
+```
+
+### `suspense/useSubscribe`
+
+This is a version of `useSubscribe` that can be used with React Suspense. 
+It is similar to `useSubscribe`, it throws a promise and suspends the rendering until the promise is resolved.
+It does not return a Meteor Handle to control the subscription
+
+```jsx
+
+import { useTracker } from 'meteor/react-meteor-data/suspense'
+import { useSubscribe } from 'meteor/react-meteor-data/suspense'
+
+function Tasks() { // this component will suspend
+  useSubscribe("tasks");
+  const { username } = useTracker("user", () => Meteor.user())  // Meteor.user() is now async
+  const tasksByUser = useTracker("tasksByUser", () =>
+          TasksCollection.find({username}, { sort: { createdAt: -1 } }).fetchAsync() // fetch will be async
+  );
+
+
+// render the tasks
+}
+
+```
+
+### `suspense/useFind`
+
+This is a version of `useFind` that can be used with React Suspense.
+It has a few differences from the `useFind` without suspense, it throws a promise and suspends the rendering until the promise is resolved.
+It returns the result and it is reactive.
+You should pass as the first parameter the collection where is being searched upon and as the second parameter an array with the arguments,
+the same arguments that you would pass to the `find` method of the collection, third parameter is optional, and it is dependency array object.
+It is intended to be used in the server(SSR)
```

Comment:
That's not entirely true, I think. It's _meant_ for the SSR, but one can use it nevertheless. And the other way round - you don't have to use it if you're not interested in SSR.